src/TuDa.CIMS.Web/Components/ShoppingCartFooter.razor.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.AspNetCore.Components;
+using MudBlazor;
+using TuDa.CIMS.Shared.Entities;
+
+namespace TuDa.CIMS.Web.Components;
+
+/// <summary>
+/// Class for the shopping cart footer
+/// </summary>
+/// <param name="dialogService"></param>
+public partial class ShoppingCartFooter(IDialogService dialogService) : ComponentBase
+{
+    /// <summary>
+    /// Parameter for the purchase
+    /// </summary>
+    [Parameter]
+    public Purchase? Purchase { get; set; }
```

Comment:
❓ Why is this nullable?

Reply:
Wasn't sure if it has to be nullable or required.